lesson02/assignment/index.html

```diff
@@ -1,15 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 02</title>
-    </head>
-    <body>
-        <main class="main">
-            <h1>title</h1>
-        </main>
-    </body>
-</html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/style.scss" />
+    <title>Assignment 02</title>
+</head>
+
+<body>
+    <main class="main">
+        <div class="content-container">
+            <div class="content-box">
+                <div class="header-box">
+                    <img src="./img/headshot.jpg" alt="dog headshot">
+                    <div class="header-title">
+                        <h2>Paolo Maldini</h2>
+                        <p>Front End Developer</p>
+                    </div>
+                </div>
+                <div class="details-box">
+                    <p><strong>Company:</strong> Web Weavers</p>
+                    <p><strong>Experience:</strong> 3 years</p>
+                    <p><strong>School:</strong> University of Washington</p>
+                    <p><strong>Major:</strong> Marketing</p>
+                    <p><strong>Email:</strong> <a href="mailto: paolo@example.com">paolo@example.com</a></p>
+                    <div class="linkedin-container">
+                        <a href="paolo.linkedinprofile.com">
```

Comment:
This href should have https:// before the rest of the link: paolo.linkedinprofile.com.  Otherwise it just links to a local host location that doesn't exist.